### ISSUE TITLE
API Fix Request (2)

### ISSUE THREAD

**Issue by JacobAMason** [henryjr1/DelterAirlinesTeam3#3]
- [x] The second GET endpoint `/flights/return` doesn't work. It seems like the optional query parameters could be used on the regular `/flights` endpoint, though, so I'm not sure why they are two separate endpoints.
 - [x] to get arrivals and departures, we can use the query that the consolidation above would provide. However, we do not use zip codes for locations. As outlined in the API specification document, we only use `City, 2-letter-state` as the format. E.g.: `Starkville, MS`

**Comment by aqd14** [comment]
There is no /flights/return endpoint. Maybe it comes from old version.

Updated to support search for City, 2-letter-state

**Comment by JacobAMason** [comment]
Here is the /flights/return endpoint
https://app.swaggerhub.com/apis/DelterArlines/DelterAirlinesAPI/1.0.0#/Flights/fromLocation_toLocation_startDate_endDate

**Comment by aqd14** [comment]
Sorry, we forgot to update the API endpoint in Swagger. I just updated. We wanted to use Flight-Search endpoint to make it's purpose clearer.

**Comment by JacobAMason** [comment]
I'll look at it. Mark this version as published and, in the future, please publish and reversion your apis on each change to insulate our team from unexpected changes

**Comment by aqd14** [comment]
Ok, thanks!

**Comment by JacobAMason** [comment]
Searches still seem to take zip codes

**Comment by aqd14** [comment]
@PBoudreaux1576 Please handle this request from Jacob

**Comment by aqd14** [comment]
@JacobAMason Please try this URL: http://35.188.55.177/api/v1.0/Flight-Search?startDate=2017-11-29&endDate=2017-11-30&fromLocation=Starkville,%20MS&toLocation=Atlanta,%20GA

**Comment by JacobAMason** [comment]
This somewhat works. Make the changes to your API on swaggerhub, which still claims to use integers as the location types. Also, if I remove the location parameters from above, the search seems to fail. Query parameters should really be optional.

**Comment by aqd14** [comment]
Thanks. Updated to use string as parameter type instead of integer. Will fix the optional parameter in short time.

**Comment by aqd14** [comment]
I don't think query parameters should be optional for flight searching. APIs consumers really need to know what they are trying to looking for when they do queries. I tried Orbiz and they also force users to input values for location, date, etc..

So, I forced the parameters with required flag is True. Also, return message if user input missing something. Now the API support both searching by date and byte datetime
